src/main/resources/edu/cornell/kfs/sys/workflow/PaymentMethodMaintenanceDocument.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<data xmlns="ns:workflow"
+	xmlns:fo="http://www.w3.org/1999/XSL/Format"
```

Comment:
I would suggest formatting this file to use spaces instead of tabs.

Reply:
updated